app/containers/gistEditorForm/index.js

```diff
@@ -1,8 +1,14 @@
 import React, { Component } from 'react'
-import { Field, FieldArray, reduxForm } from 'redux-form'
+import { connect } from 'react-redux'
+import { Field, FieldArray, reduxForm, formValueSelector } from 'redux-form'
 import { Button, ListGroup, ListGroupItem, Panel } from 'react-bootstrap'
 import CodeMirror from 'react-codemirror'
+import { modeInfo } from './meta'
+import 'codemirror/mode/xml/xml'
 import 'codemirror/mode/markdown/markdown'
+import 'codemirror/mode/clike/clike'
+import 'codemirror/mode/javascript/javascript'
+// TODO: list all mode here ?
```

Comment:
For now, I have to import all mode files in order to support all syntax highlighting ...